addons/website_quote/views/website_quotation.xml

```diff
@@ -507,6 +507,11 @@
                         </div>
                     </div>
                     <div class="col-md-9">
+                        <div class="row">
+                            <div groups="base.group_website_publisher" t-ignore="true" class="css_editable_mode_hidden col-md-offset-11">
```

Comment:
@tde-banana-odoo Back button only visible to the login user, but controller have  auth="user". so no require groups, can i remove it?